### PR TITLE
Adds fingerprint transfer for machine deconstruction; logs SMES

### PR DESCRIPTION
**Pull request by harmonyn**
Fixes https://github.com/tgstation/-tg-station/issues/10413.
- All machine deconstruction transfers fingerprints to the machine frame created.
- SMES has logging for the player who deconstructed it, as well as singulo logging. No longer can disgruntled CEs "hold the power hostage" undetected.

### CODE REVIEW

File: code/modules/power/smes.dm [hunk 1]
```diff
@@ -159,6 +159,9 @@
 
 	//crowbarring it !
 	default_deconstruction_crowbar(I)
+	message_admins("[src] has been deconstructed by [key_name_admin(user)](<A HREF='?_src_=holder;adminmoreinfo=\ref[user]'>?</A>) (<A HREF='?_src_=holder;adminplayerobservefollow=\ref[user]'>FLW</A>) in ([x],[y],[z] - <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[x];Y=[y];Z=[z]'>JMP</a>)",0,1)
+	log_game("src] has been deconstructed by [key_name(user)]")
```
Comment (ike709) [inline code/modules/power/smes.dm]: Pretty sure this is a typo and should be [src] not src]

Comment (RemieRichards) [inline code/modules/power/smes.dm]: yep.

Comment (bgobandit) [inline code/modules/power/smes.dm]: honk

**Comment by Shadowlight213** [comment]
Based

**Comment by Stickymayhem** [comment]
I'll be adding a PR which automatically detonates eborgs who are being shit soon <3

**Pull request merged**